supervision/detection/vlm.py

```diff
@@ -315,3 +350,64 @@ def from_florence_2(
         return xyxy, labels, None, None
 
     assert False, f"Unimplemented task: {task}"
+
+
+def from_google_gemini(
+    result: str,
+    resolution_wh: Tuple[int, int],
+) -> Tuple[np.ndarray, np.ndarray]:
```

Comment:
This API is inconsistent with the `from_paligemma` and `from_qwen_2_5_vl` implementations.

Currently, it is not possible to resolve `class_id` values. To address this, we should allow users to optionally provide a `classes: Optional[List[str]] = None` argument. If this argument is given, we should attempt to resolve the `class_id` for each detection, following the same approach as in `from_paligemma` and `from_qwen_2_5_vl`.

The function should return `Tuple[np.ndarray, Optional[np.ndarray], np.ndarray]`, consistent with the return type of `from_paligemma` and `from_qwen_2_5_vl`, where `Optional[np.ndarray]` corresponds to `class_id`. This value should be `None` if `classes` is not provided, and an `np.ndarray` if it is.